test/libsolidity/smtCheckerTests/types/array_static_aliasing_memory_5.sol

```diff
@@ -1,9 +1,6 @@
 contract C
 {
 	function f(uint[2] memory a, uint[2] memory b, uint[2] memory c) public pure {
-		require(a.length > 0);
-		require(b.length > 0);
-		require(c.length > 0);
```

Comment:
Why remove these? To ensure this case tests aliasing only?

Reply:
They're redundant since the type ensures that length == 2